org.phoenicis.playonlinux.yml

```diff
@@ -0,0 +1,67 @@
+---
+app-id: org.phoenicis.playonlinux
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+add-extensions:
+  # support 32-bit apps
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: 18.08
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: 18.08
+    no-autodownload: true
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: 18.08
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+command: phoenicis-playonlinux
+finish-args:
+  # restrict filesystem access to .Phoenicis
+  # and the download and documents directories (for local installers)
+  - --filesystem=~/.Phoenicis
```

Comment:
Isn't it saner to use `--persist=.Phoenicis`?

Reply:
I'm not sure if I understood the discussion. Are applications started outside the flatpak environment somehow?

Reply:
I'll try to clarify a bit more: Phoenicis creates regular Wine prefixes. So, even though it creates them, I would not consider that it owns them. For example, you could also start the application with your system Wine or any other GUI which allows you to do so. You could even install Phoenicis via .deb and via flatpak. Therefore I don't think it makes sense (neither from a technical nor a logical perspective) to put it inside the flatpak environment.

Reply:
@AsciiWolf 
> I just found out that the `--filesystem=~/.Phoenicis` doesn't work. :-( I think that it doesn't like `~/` at the beginning, unfortunately I don't know what to use instead of this. The best approach would be using `--persist` instead.

It should work. Maybe you should add the `:create` postfix?

See also:
http://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#filesystem-permissions

Reply:
Seems to work fine after adding the `:create` suffix! Thanks!

Reply:
Just to clarify: Do you want me to add this to the yaml?

Reply:
Yes. (For _~/.Phoenicis_, it is not needed for the xdg-* dirs.) It doesn't work without the suffix.

Reply:
Sure.

Reply:
Does it work now?

Reply:
Works fine, thanks!